### PR TITLE
UnEscapeIndicatorIPv6 script fixes

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29012

## Description
Fixed an issue where the script needs to extract the IPv6 address from specific characters.
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ronykoz — COMMENTED** [review]
Add more tests to the TPB, i want to cover all cases

File: Packs/CommonScripts/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.2.63",
+    "currentVersion": "1.2.70",
```
Comment (ronykoz) [inline Packs/CommonScripts/pack_metadata.json:5]: why not 64?

**Review by ronykoz — COMMENTED** [review]
Let's go over the TPB together please

File: Packs/CommonScripts/pack_metadata.json (hunk 1 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/pack_metadata.json:5]: just for now, to avoid conflicts

**Review by ronykoz — APPROVED** [review]

**Pull request merged**